File: 2015/1-not-quite-lisp/day1p1.py
filename = '../source/day1source.txt'

with open(filename) as f:
    source = [line.rstrip('\n') for line in open(filename)]

floor = 0

for line in source:
    for char in line:
        if char == '(':
            floor = floor + 1
        else:
            floor = floor - 1

print "Santa ends up on floor", floor

File: 2015/01-not-quite-lisp/day1p1.js
var fs = require('fs')

var source = fs.readFileSync('./input.txt').toString()
var end = source.length
var floor = 1
for (var i = 0; i < end; i++) {
  if (source.charAt(i) === '(') {
    floor++
  } else {
    floor--
  }
}
console.log('Santa ends up on floor ' + floor + '.')
